### PR TITLE
ProgressBars - make dynamic

### PR DESCRIPTION
**Pull request by s-cork**

### CODE REVIEW

File: client_code/ProgressBar/__init__.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
   content: '';
   -webkit-animation: running-progress 2s cubic-bezier(0.4, 0, 0.2, 1) infinite;
   animation: running-progress 2s cubic-bezier(0.4, 0, 0.2, 1) infinite;
+  background-color: var(--anvil-extras-track-colour);
```
Comment (meatballs) [inline client_code/ProgressBar/__init__.py:42]: always nice to see a css variable in the wild!!

**Comment by meatballs** [comment]
Can we get a changelog entry for this one?

**Comment by s-cork** [comment]
Can - shall i change log it and merge?

**Comment by meatballs** [comment]
I'm around this morning. Happy to merge and release in a wee while...

**Pull request merged**